frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -124,9 +127,10 @@ const MonitoringAlerts: React.FC<MonitoringAlertsProps> = () => {
   const dispatch = useDispatch();
   const rule = useSelector((state: RootState) => state.UI.getIn(['monitoring', 'devRules']));
 
-  // fetch and store the rules in store
+  // fetch and store the alerts & rules in store
   React.useEffect(() => {
     dispatch(monitoringSetRules('devRules', getAlertsAndRules(rules?.data).rules));
+    dispatch(monitoringLoaded('alerts', getAlertsAndRules(rules?.data).alerts));
```

Comment:
Dev Alert should store in different redux store under monitoring. Otherwise, it will break the admin alert page.

<img width="1459" alt="Screenshot 2020-06-26 at 11 25 29 AM" src="https://user-images.githubusercontent.com/2561818/85826464-40ebc680-b7a2-11ea-84de-bbf1ab26add5.png">

Reply:
Done